nautobot/core/celery/log.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from celery import current_task
+
+
+class NautobotDatabaseHandler(logging.Handler):
+    """Custom logging handler to log messages to JobLogEntry database entries."""
+
+    def emit(self, record):
+        if current_task is None:
+            return
+
+        from nautobot.extras.models.jobs import JobResult
+
+        try:
+            self.format(record)
+
+            if record.task_id == "???":
+                return
+            job_result = JobResult.objects.filter(id=record.task_id)
+            if not job_result.exists():
+                return
```

Comment:
Are there cases where `???` would be frequent enough to not do the search always? Seems redundant.

Reply:
We would have to wrap this in a try/except block because if you pass a string that's not a uuid to this filter it raises an exception: `django.core.exceptions.ValidationError: ['“???” is not a valid UUID.']`

Reply:
I think a try/except might be more robust and easier to read.